pygmt/src/plot.py

```diff
@@ -6,6 +6,7 @@
 from pygmt.helpers import (
```

Comment:
FYI, I plan to revert the changes in `plot.py` before merging this PR, because the `deprecate_parameter` decorator is already tested in its docstrings, and the deprecation of `sizes` parameter need more changes and should be done in a separate PR.